lab-bulma-react/src/App.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+
+// Navbar
+
+function Navbar() {
+  return (
+<nav class="navbar is-transparent">
+  <div class="navbar-brand">
```

Comment:
Remember in react, is `className` not `class`. You should see some warnings from react